libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.h

```diff
@@ -140,4 +140,5 @@ class AP_InertialSensor_Invensensev3 : public AP_InertialSensor_Backend
 
     float temp_filtered;
     LowPassFilter2pFloat temp_filter;
+    uint32_t sampling_rate_hz;
```

Comment:
Where is this used other than in the boot message?

Reply:
Yes, that's it. Its used to convey that backend rate is different than the actual sampling rate